tasks/zip.js

```diff
@@ -10,15 +10,31 @@ module.exports = function( gulp ) {
 	// this task copies files we'll zip into a build directory
 	gulp.task( 'zip-copy-files', function() {
 		let packageContents = fs.readFileSync( './package.json', 'utf8' );
-		let packageWhitelistContents = fs.readFileSync( './package-whitelist.json', 'utf8' )
+		let packageSafeListContents;
+
+		try {
+			if ( fs.accessSync( './package-safelist.json', fs.constants.F_OK ) ) {
+				packageSafeListContents = fs.readFileSync( './package-safelist.json', 'utf8' )
+			} else {
+				packageSafeListContents = fs.readFileSync( './package-whitelist.json', 'utf8' )
+			}
+
+		} catch( e ) {
+			console.log( e );
+		}
+
 		let json = parseJson( packageContents );
-		let zipInclude = parseJson( packageWhitelistContents );
+		let zipInclude = parseJson( packageSafeListContents );
 		let commonZipContents;
 
 		try {
-			commonZipContents = fs.readFileSync( './common/package-whitelist.json', 'utf8' );
+			if ( fs.accessSync( './common/package-safelist.json', fs.constants.F_OK ) ) {
+				commonZipContents = fs.readFileSync( './common/package-whitelist.json', 'utf8' );
```

Comment:
Copy pasta? Should be?
```suggestion
				commonZipContents = fs.readFileSync( './common/package-safelist.json', 'utf8' );
```